harvester_e2e_tests/integration/test_backup_restore.py

```diff
@@ -313,6 +351,7 @@ def test_restore_with_new_vm(
         with host_shell.login(host_ip, jumphost=True) as h:
             vm_sh = vm_shell(base_vm_with_data['ssh_user'], pkey=pri_key)
             endtime = datetime.now() + timedelta(seconds=wait_timeout)
+            login_ex = None

```

Comment:
Could you explain the case about `login_ex` is not defined?
I think it is not possible as we will encountered at least once before
```python
raise AssertionError(...) from login_ex
```

otherwise we will never hit the timeout of while loop :thinking:

Reply:
I think it is lint's bug, when `ChannelException` not hit, we will `break` or encountered another exception, then the while loop will never execute `else` closure.

I guess the bug of lint might be, they will not detect the `else` closure of try-except, thus it will looks like the variable is unbound.

Reply:
FYI: https://github.com/microsoft/pylance-release/issues/2507#issuecomment-1077662362

Reply:
Reverted warning fix in latest commit.